### PR TITLE
Add code snippet to add a currency and symbol

### PR DESCRIPTION
**Pull request by nielslange**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woodocs/issues/100

This PR adds the code snippet [Add a currency and symbol](http://www.woocommerce.com/document/add-a-custom-currency-symbol/) to the Woo docs.

### How to test the changes in this Pull Request:

1. Compare the content of the file docs/snippets/add-a-currency-symbol.md with the content on http://www.woocommerce.com/document/add-a-custom-currency-symbol/.
2. Verify that the new link within docs/snippets/README.md points to the newly added code snippet.

### CODE REVIEW

**Review by barryhughes — APPROVED** [review]
Looks good, confirmed the snippet continues to work as expected.

**Comment by barryhughes** [comment]
One of the required checks failed, but it's clearly flakiness. Triggered a re-run and we can merge as soon as it passes.

File: docs/snippets/add-a-currency-symbol.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Add a currency and symbol
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
```
Comment (barryhughes) [inline docs/snippets/add-a-currency-symbol.md]: Sorry, further piece of feedback. Per Brent, we can and should drop these developer-level doc warnings.

Comment (nielslange) [reply docs/snippets/add-a-currency-symbol.md]: Removing this note based on p1692723749470709/1692723559.960259-slack-C0516GKR69W

**Pull request merged**